### PR TITLE
The Base HTML

### PR DESCRIPTION
**Pull request by Lane-Dillard**
What?
This is the base HTML for the repository.
Why?
To build the base of what is needed for the website,
How?
This should be able to make the "logo" and the title within the header of it, with the needed DOM function in main which we will do in the near future, and the contact information in the footer
Testing?
To make sure it is all good with everyone's taste on how it was done.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <body>
+        <main id = "container">
+            //Where the DOM variable from main.js will go
+
+        </main>
+
+        <script type="module" src="./scripts/main.js"></script>
+    </body>
+
+    <footer>
```
Comment (zsazsazsa) [inline index.html:22]: The footer should be before the close of the body, just before the scripts

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Parks</title>
+        <header>
```
Comment (zsazsazsa) [inline index.html:6]: Header should be inside the body element

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            <img src="https://i.pinimg.com/736x/e0/b3/56/e0b35696d26ad189cb3301364c1bada1.jpg" alt="Header Image">
+            <h1>Cider Falls</h1>
+        </header>
```
Comment (Caleb-Ford-93) [inline index.html:9]: Need to move header element down to first line inside body element i.e. header should start on line 14

**Review by zsazsazsa — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        </main>
+
+        <script type="module" src="./scripts/main.js"></script>
+    </body>
+
+    <footer>
+        <p>Email Address: -------------------------</p>
+        <p>Street Address: ----------------------</p>
+        <p>Phone Number: ----------</p>
+    </footer>
```
Comment (Caleb-Ford-93) [inline index.html:26]: Same as header, footer element should be right above script tags inside body tag

**Review by Caleb-Ford-93 — CHANGES_REQUESTED** [review]
Mostly looks good, just a couple formatting changes

File: index.html (hunk 4 above)
Comment (Lane-Dillard) [reply index.html]: It should be updated now

**Review by Caleb-Ford-93 — APPROVED** [review]
Looks good to me! you are go for merge!

**Review by zsazsazsa — APPROVED** [review]
Looks good!

**Pull request merged**